### PR TITLE
Update to work with LilyPond 2.24

### PR DESCRIPTION
**Pull request by yurrriq**
Thanks for this project! I had to fork it to get it working with LilyPond 2.24.1 and figured I'd contribute back. I'm happy to apply any feedback.

**Edit:** I've applied @jeanas's [corrections](https://github.com/openlilylib/gridly/pull/3#discussion_r1301137513).

### CODE REVIEW

File: package.ily [hunk 1]
```diff
@@ -37,12 +37,10 @@
 #(define gridly-version "0.6.0")
 
 %%% The association list holding all the music.
-#(if (not (defined? 'music-grid))
```
Comment (yurrriq) [inline package.ily:40]: LilyPond was complaining about this for some reason...

Comment (jeanas) [reply package.ily:40]: See https://lilypond.org/doc/v2.24/Documentation/changes/index#notes-on-guile-2_002e2

Comment (yurrriq) [reply package.ily:40]: Thanks! I'll plan to amend in the next few days.

**Review by jeanas — APPROVED** [review]
Looks good, thank you!

**Pull request merged**